docs/Software Architecture Document/Components View/index.md

```diff
@@ -229,6 +245,27 @@ The view is represented by module and subsystem diagrams that show the system's
   - *Constraints*
     - [SmartDeviceLink Protocol specification](https://github.com/smartdevicelink/protocol_spec/blob/master/README.md)
 
+#### Remote control
+  - *Responsibility*
+    - Allows incorporating additional functionality to the core application by application extension.
+    - Implements specific mobile RPC handling.
+    - Implements specific HMI RPC handling.
+  - *Relations*
+    - Composed by ***Plugin manager*** 
+    - Handles ***Application Manager*** by ***Service*** interface
+  - *Interfaces*
+    - Provides ***Plugin Manager*** interface   
+  - *Behavior*
+    - Receives data from CoreService
+    - Parses data
+    - Creates commands.
+    - Handles incoming HMI notifications
+    - Sends RPC to HMI
+    - Sends RPC to mobile
+    - Extends basic applications with additional RPC's
+  - *Constraints*
+    - N/A
```

Comment:
@LuxoftAKutsan, 
Requirments are the base of Architecture. Architecture could not be constructed and designed  without it.